docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,27 @@
 title: Changelog
 ---
 
+## 13.10.0
+
+_Released 5/21/2024_
+
+**Features:**
+
+- Added support for `vite` `v5` to `@cypress/vite-dev-server`. Addresses [#28347](https://github.com/cypress-io/cypress/issues/28347).
```

Comment:
Do we want to link out to vite v5?

Reply:
like the blogpost? https://vitejs.dev/blog/announcing-vite5

Reply:
added the blogpost ref in a0e6c6bc6